pkgs/development/python-modules/future-stubs/002-add-py.typed-files.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/MANIFEST.in b/MANIFEST.in
+index 46551ba..cc5b512 100644
+--- a/MANIFEST.in
++++ b/MANIFEST.in
+@@ -3,3 +3,5 @@ prune tests
+ prune stubs
+ recursive-exclude src *.py
+ recursive-include src *.pyi
++include src/future/py.typed
++include src/past/py.typed
```

Comment:
not a blocker for this PR, but this should probably be upstreamed. Nothing nixpkgs specific about it.